src/js/stores/AppsStore.js

```diff
@@ -134,6 +134,7 @@ function processApp(app) {
   app.healthWeight = getAppHealthWeight(app.health);
 
   app.tasks = lazy(app.tasks).map(function (task) {
+    task.id =  task.id || task.taskId;
```

Comment:
I've added this to the store in case `task.id` is some how undefined, but removed all similiar checks within the components.